theme/tutorial.less

```diff
@@ -471,6 +471,50 @@ code.lang-filterblocks {
     margin: 0.5em;
 }
 
+
+/******************************
+    Immersive Reader button
+******************************/
+#mainmenu .immersive-reader-button.ui.item,
+#mainmenu .immersive-reader-button.ui.item:focus,
+.hc #mainmenu .immersive-reader-button.ui.item:focus {
```

Comment:
are the high contrast/focus selectors here necessary? will the first one cover those cases?

Reply:
oh yeah in high contrast the background gets overridden by the high contrast rules -- I can try to add !important here instead if that's better

Reply:
does the high contrast case below fix the background issue? or does it get overriden by the focus rules?

Reply:
the case below does fix until it gets focused -- then it turns black

Reply:
oh yeah maybe just add the :focus to the one below, then? just so all the overriding is happening in one place